main/test/src/util/ScriptTestSuite.scala

```diff
@@ -79,8 +80,17 @@ abstract class ScriptTestSuite(fork: Boolean) extends TestSuite {
     }
   }
   private def evalFork(stdout: os.ProcessOutput, s: Seq[String]): Boolean = {
+    val millRelease = Option(System.getenv("MILL_TEST_RELEASE"))
+      .getOrElse(throw new NoSuchElementException(
+        s"System environment variable `MILL_TEST_RELEASE` not defined. It needs to point to the Mill binary to use for the test."
+      ))
```

Comment:
```suggestion
    val millRelease = sys.env.getOrElse("MILL_TEST_RELEASE", throw new NoSuchElementException(
      "System environment variable `MILL_TEST_RELEASE` not defined. It needs to point to the Mill binary to use for the test."
    ))
```

Reply:
`sys.env` retrieves the whole env into a Map and handles missing elements by a second case-insensitive lookup, etc. My motivation was to avoid doing unnecessary things here.

Reply:
Makes sense. 👍 Thank you for writing down the motivation.